src/loaders/scraper_browse_id.py

```diff
@@ -15,10 +16,14 @@ def save_browse_page(content: str, page: int) -> None:
         page (int): page number to identify file with
     '''
     filename = f'{config.DATA_PATH}/raw/bgg_browse_page_{str(page)}.html'
```

Comment:
Since you have created a new `pathname` variable, lets strip the `filename` variable to just the filename and we can combine them when we need to use the full path.